tests/installation/bootloader_s390.pm

```diff
@@ -301,4 +302,25 @@ sub run {
     $self->result('ok');
 }
 
+sub post_fail_hook {
+    my $s3270 = console('x3270');
+    my $r;
+
+    if (check_screen 'linuxrc') {
+        # Start linuxrc shell
+        $s3270->sequence_3270(qw(String("x") ENTER String("3") ENTER));
+        assert_screen 'linuxrc-shell';
+
+        # collect linuxrc logs
+        #$s3270->sequence_3270((String(\"cat /var/log/linuxrc.log\"), ENTER));
+        $s3270->sequence_3270("String(\"cat /var/log/linuxrc.log && echo 'LINUXRC_LOG_SAVED'\")", "ENTER",);
+
+        $r = $s3270->expect_3270(
+            output_delim => qr/LINUXRC_LOG_SAVED/,
+            timeout      => 60
+        );
+        $r ? record_info 'Logs collected', 'Linuxrc logs can be found in autoinst-log.txt' : die "Could not save linuxrc logs";
```

Comment:
I'n not sure how exactly it sequence_3270 works, but will it be in autoinst log, or in serial or in both?

Reply:
only in autoinst-log.txt